src/linux/elf.rs

```diff
@@ -397,6 +353,72 @@ impl ElfInfo {
             stack,
         })
     }
+
+    pub fn merge(left: ElfInfo, right: ElfInfo) -> ElfInfo {
```

Comment:
I must admit I'm a bit stumped by this. I thought that symbolic would already have functionality to present a single view of a PE+PDB or .so+.dbg couple. It's a bit sad to see we have to merge the debuginfo ourselves.